docs/source/reading_writing.rst

```diff
@@ -96,13 +95,18 @@ class: ``Samples``, ``MCMCSamples``, or ``NestedSamples``.
       from anesthetic import Samples  # or MCMCSamples, or NestedSamples
       samples = Samples(read_csv("filename.csv"))
 
-* ``read_parquet``:
+When loading in previously saved samples from hdf5, make sure to import the
+``anesthetic.io.read_hdf`` function, and not the ``pandas.read_hdf`` version. If
+you forget to do this, the samples will be read in as a ``DataFrame``, with a
+consequent loss of functionality
+
+
+* ``read_hdf``:
 
   ::
   
-      from pandas import read_parquet
-      from anesthetic import Samples  # or MCMCSamples, or NestedSamples
-      samples = Samples(read_parquet("filename.parquet"))
+      from anesthetic.io import read_hdf
+      samples = read_hdf("filename.h5", "samples")
```

Comment:
In the interest of being similar to both `read_chains` and to pandas, I'd say it should be possible to import `read_hdf` directly from `anesthetic`, i.e.:
```python
from anesthetic import read_hdf
```

Reply:
done in 9426d0b